src/main/java/com/google/maps/ElevationApi.java

```diff
@@ -32,6 +33,10 @@
  */
 public class ElevationApi {
   private static final String BASE = "/maps/api/elevation/json";
+  private static final ApiConfig<ElevationResult> SINGLE_API_CONFIG =
+      new ApiConfig<ElevationResult>(SingularResponse.class, BASE);
+  private static final ApiConfig<ElevationResult[]> MULTI_API_CONFIG =
+      new ApiConfig<ElevationResult[]>(MultiResponse.class, BASE);
```

Comment:
I wonder if it's worth moving the response type out of the config. It's not part of the config for a certain API, but rather a particular API call, right?